### PR TITLE
#166630607 Fix password reset

### PR DESCRIPTION
**Pull request by Philipotieno**
#### What does this PR do?
- Fix password reset and confirm password
#### Description of Task to be completed?
- Disable reset and submit button before valid details are entered
#### How should this be manually tested?
- `$ git clone https://github.com/andela/ah-the-immortals-frontend`
- `$ git checkout bg-fix-password-reset-166630607`
- `$ npm install`
- `$ npm run start:dev`
#### What are the relevant pivotal tracker stories?
[`#166630607`](https://www.pivotaltracker.com/n/projects/2326460/stories/166630607)
#### Screenshots
[http://recordit.co/4Dsr7mI0QW](http://recordit.co/4Dsr7mI0QW)
image

image
image

### CODE REVIEW

**Review by IssaIan — APPROVED** [review]
Good work on implementing the feedback I had given @Philipotieno.

**Review by matthenge — APPROVED** [review]
LGTM

File: src/views/password_reset/__tests__/PasswordResetConfirmContainer.test.js [hunk 1]
```diff
@@ -52,4 +53,19 @@ describe('Tests PasswordResetConfirmContainer', () => {
...
+    const message = form.find('[className="matching"]');
+    // const input = {
+    //   password_confirm:form.find('input[name="password_confirm"]')
+    // };
+    Object.values(inputField).forEach(input => {
+      input.simulate('keyup', {
+        keyCode: 65
+      });
+    });
+    console.log(message.debug());
```
Comment (Ogutu-Brian) [inline src/views/password_reset/__tests__/PasswordResetConfirmContainer.test.js]: Hello @Philipotieno ,
Please remove the console statement

File: src/styles/style.css [hunk 2]
```diff
@@ -166,3 +166,45 @@ label {
...
+    width: 358px;
+    padding: 5px;
+    z-index: 5;
+    font-size: 25px;
+    font-weight: bolder;
+    margin:0 auto;
+    color: #f7f7f7;
+    text-decoration: none;
+    margin-right: 10px
+}
```
Comment (Ogutu-Brian) [inline src/styles/style.css]: Hello,
Please add a new line at the end of the file

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

File: src/views/password_reset/__tests__/PasswordResetConfirmContainer.test.js (hunk 1 above)
Comment (Philipotieno) [reply src/views/password_reset/__tests__/PasswordResetConfirmContainer.test.js]: Done

**Review by Ogutu-Brian — APPROVED** [review]

**Pull request merged**